Libraries/Components/ScrollView/ScrollView.js

```diff
@@ -497,6 +498,10 @@ const ScrollView = React.createClass({
       props.decelerationRate = processDecelerationRate(decelerationRate);
     }
 
+    if (Platform.OS === 'android' && props.snapToInterval) {
```

Comment:
This shouldn't happen here.  It should be adjusted in native code like we do all other measures